src/features/session_replay/shared/recorder.js

```diff
@@ -99,13 +99,6 @@ export class Recorder {
       inlineImages: inline_images,
       collectFonts: collect_fonts,
       checkoutEveryNms: CHECKOUT_MS[this.parent.mode],
-      /** Emits errors thrown by rrweb directly before bubbling them up to the window */
```

Comment:
This removal needs to be mentioned in the PR description + have relevant ticket linked as it's an impactful change